test/integration/suites/batchx509SVID/02-create-entries.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+ENTRIES_FILE="${RUNDIR}/data.json"
+
+# Start the entry creation process in the background
+spire-server entry create -data "${ENTRIES_FILE}" &
+ENTRY_PID=$!
+
+# Wait for the entry creation process to finish
+wait $ENTRY_PID
```

Comment:
afaik the other CI don't do this PID monitoring, and instead look for success of the CLI command and/or look for success logs

Reply:
we could follow the existing retry pattern present throughout, like at https://github.com/spiffe/spire/blob/main/test/integration/suites/fetch-x509-svids/04-create-registration-entries

again, for consistency; if this is a pattern that is deemed better than we should propagate that one to the rest of CI rather than have divergent patterns which may either confuse future contributors about what the standard is or make them think there is no standard at all and they can introduce a 3rd pattern